core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -101,12 +109,25 @@ public void commit(TableMetadata base, TableMetadata metadata) {
       return;
     }
 
-    doCommit(base, metadata);
+    Stopwatch stopwatch = Stopwatch.createStarted();
+    String newMetadataLocation = doCommit(base, metadata);
     deleteRemovedMetadataFiles(base, metadata);
     requestRefresh();
+    stopwatch.stop();
+
+    LOG.info("Successfully committed to table: {} in: {} ms: {}",
```

Comment:
I don't think we need so many colons. How about `Successfully committed to table {} in {} ms`?

Reply:
Done